### ISSUE TITLE
Commissioner HTML Documents

### ISSUE THREAD

**Issue by patphongs** [18F/fec-cms#616]
Below is a list of HTML files that are part of former Commissioner's documents. We need to figure out how to migrate these over. Some documents are  Statements of Reasons, which should have an equivalent PDF file within the EQS system. Those statements should be identified. The rest of the documents can probably be converted to either PDF documents or a static HTML file can be created inside of Wagtail in its place. @emileighoutlaw @LindsayYoung

@AmyKort We have to identify a resource on the FEC side who is a knowledge matter expert to help us identify the documents within EQS?

http://www.fec.gov/members/former_members/toner/opinions/opinions20060926.shtml
http://www.fec.gov/members/former_members/toner/opinions/opinions20060314.shtml
http://www.fec.gov/members/former_members/toner/opinions/opinions20051207.shtml
http://www.fec.gov/members/former_members/toner/opinions/opinions20050615.shtml
http://www.fec.gov/members/former_members/thomas/thomasarticle2.htm
http://www.fec.gov/members/former_members/thomas/thomasarticle1.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement12.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement09.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement08.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement07.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement06.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement05.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement04.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement03.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement02.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement01.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement14.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement13.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement15.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement16.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement17.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement18.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement19.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement39.html
http://www.fec.gov/members/former_members/thomas/thomasstatement38.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement37.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement36.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement35.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement34.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement33.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement32.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement31.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement30.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement28.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement27.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement26.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement23.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement22.htm
http://www.fec.gov/members/former_members/thomas/thomasstatement40.html
http://www.fec.gov/members/former_members/thomas/thomasstatement41.html
http://www.fec.gov/members/former_members/thomas/thomasstatement43.html
http://www.fec.gov/members/former_members/thomas/thomasstatement42.html
http://www.fec.gov/members/former_members/thomas/thomasstatement44.html
http://www.fec.gov/members/former_members/thomas/thomasstatement45.html
http://www.fec.gov/members/former_members/smith/smithstatement04.html
http://www.fec.gov/members/former_members/smith/smithstatement03.html
http://www.fec.gov/members/former_members/smith/smithstatement02.html
http://www.fec.gov/members/former_members/smith/smithstatement01.html
http://www.fec.gov/members/former_members/smith/smithspeech02.html
http://www.fec.gov/members/former_members/smith/smithspeech01.html
http://www.fec.gov/members/former_members/smith/smithspeech.shtml
http://www.fec.gov/members/former_members/smith/smithreason6.htm
http://www.fec.gov/members/former_members/smith/smithreason5_files/header.htm
http://www.fec.gov/members/former_members/smith/smithreason5.html
http://www.fec.gov/members/former_members/smith/smithreason4.html
http://www.fec.gov/members/former_members/smith/smithreason3.html
http://www.fec.gov/members/former_members/smith/smithreason2.html
http://www.fec.gov/members/former_members/smith/smithreason1.html
http://www.fec.gov/members/former_members/smith/smithreason07.html
http://www.fec.gov/members/former_members/smith/smithopinions.shtml
http://www.fec.gov/members/former_members/smith/smithinfo_bcra.shtml
http://www.fec.gov/members/former_members/smith/smithcomment2.html
http://www.fec.gov/members/former_members/smith/smithcomment1.html
http://www.fec.gov/members/former_members/mason/masonstatement5.htm
http://www.fec.gov/members/former_members/mason/masonstatement4.htm
http://www.fec.gov/members/former_members/mason/masonstatement3.htm
http://www.fec.gov/members/former_members/mason/masonstatement2.htm
http://www.fec.gov/members/former_members/mason/masonstatement1.htm
http://www.fec.gov/members/former_members/mason/masonspeech2.htm
http://www.fec.gov/members/former_members/mason/masonspeech1.htm
http://www.fec.gov/members/former_members/mason/masonspeech.shtml
http://www.fec.gov/members/former_members/mason/masonopinions.shtml
http://www.fec.gov/members/former_members/mason/masonconcur1.htm
http://www.fec.gov/members/former_members/mason/masonarticle3.htm
http://www.fec.gov/members/former_members/mason/masonarticle2.htm
http://www.fec.gov/members/former_members/mason/masonarticle1.htm

**Comment by noahmanger** [comment]
Are these the same type of thing as the other PDFs are? If so, I don't have a huge problem with making them PDFs for consistency. But then again, it feels weird to turn HTML into PDFs. 

If we want to keep them HTML, I'd say the simplest thing would be to just copy and paste into pages in Wagtail. With only ~75 pages, it wouldn't take long at all (almost certainly faster than building a scraper / importer).

**Comment by emileighoutlaw** [comment]
I also feel weird about turning HTML pages into PDFs. Although there's something appealing about the consistency of having _all_ PDFs, the styles among the existing Commissioner PDFs are already inconsistent. 

For example: Here's [one style of PDF](http://www.fec.gov/members/statements/Goodman_AO_2015-03_Democracy%20Rules.pdf) and [here's another](http://www.fec.gov/members/weintraub/testimony/testimony20031016.pdf). 

So I don't think having all PDFs would particularly be a huge win for consistency or providing visual clues to the user that they're looking at the same type of document across statements.  

**tl;dr**: I think we should keep the HTML.

If we're keeping the HTML pages as HTML, I think the `press release` template might be the closest fit, because it has an author field— an important feature for statements.

I also agree that manual entry of those pages into Wagtail is likely the simplest and most time-effective solution. I'm happy to scrub in on this, if that's how we choose to move forward.

**Comment by noahmanger** [comment]
Good call on using the press release template! I think that could work. We'd probably want to clone the template first.

Question on how these are used: are these also linked to from our AO / MUR records? I see that many are statements in those cases.

**Comment by emileighoutlaw** [comment]
Oh that's an excellent, excellent question. 

Is there a way for our devs (Vraj or Tony or Tadhg?) to see if any MURs or advisory opinions reference HTML statements?

**Comment by patphongs** [comment]
@emileighoutlaw @noahmanger: According to @AmyKort, only Statements of Reasons would be included in EQS. Concurring or dissenting statements on AOs would be within SAOs. Still getting confirmation, but if the documents below are indeed equivalent, we can link to the already migrated legal document within the legal S3 bucket rather than creating a new HTML page for it. Anything else that we can't find within the legal documentation in the S3, I'd say we could create a static HTML page. We can discuss further today.

http://www.fec.gov/members/former_members/thomas/thomasstatement12.htm
http://saos.fec.gov/aodocs/1007086.pdf
https://cg-191800bb-c3e9-42e8-bbe9-66c882ad858f.s3.amazonaws.com/legal/aos/66472.pdf

**Comment by patphongs** [comment]
@emileighoutlaw @noahmanger @AmyKort: We're finished identifying new URLs for specific HTML files from formal commissioners. These files are marked inside of the content spreadsheet Commissioner's tab. Approximately half have an existing equivalent PDF file and the other half need to be converted into CMS pages.

**Comment by noahmanger** [comment]
Great. So the plan is:
- When a PDF of a statement exists (because it is associated with some case) we will link to that instead
- If there's no PDF, we will copy and paste into a new HTML page. This work will happen next sprint when we're moving over all of the pages in this section.

Closing.
